pkg/apiserver/api_installer.go

```diff
@@ -944,6 +948,11 @@ func addObjectParams(ws *restful.WebService, route *restful.RouteBuilder, obj in
 			}
 			switch sf.Type.Kind() {
 			case reflect.Interface, reflect.Struct:
+			case reflect.Ptr:
+				if sf.Type.Elem().Kind() == reflect.Interface || sf.Type.Elem().Kind() == reflect.Struct {
```

Comment:
you can add something like

```
// TODO... (add appropriate message here to say this is a hack and need to be fixed in a generic way later, hopefully with a bug number)
if (...) && strings.TrimPrefix(sf.Type.String(),"*") != "unversioned.Time"
```

 to let unversioned.Time pass through and have the removed time field back to the spec.